net-nntp/suck/suck-4.3.3.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools
+
+DESCRIPTION="Grab news from a remote NNTP server and feed them to another"
+HOMEPAGE="https://lazarus-pkgs.github.io/lazarus-pkgs/suck.html"
+SRC_URI="https://github.com/lazarus-pkgs/suck/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="ssl perl"
+
+DEPEND="sys-libs/db
+	perl? ( dev-lang/perl )
+	ssl? ( dev-libs/openssl )"
+
+PATCHES=( "${FILESDIR}/${PV}-fputs.patch" )
+
+src_prepare() {
+	# Fix paths to the locations in Gentoo
+	sed -i \
+		-e 's:/usr/bin/rnews:/usr/lib/news/bin/rnews:' \
```

Comment:
Is this using `rnews` from `net-nntp/inn`? That package seems to install the binary to  `/usr/$(get_libdir)/news/$(get_libdir)`, so the `sed` should do the same (hardcoded lib is almost always wrong). `net-nntp/inn` is probably missing in `RDEPEND` as `lpost` seems useless without it. This would introduce a runtime-only dependency, usually if there are both DEPEND- and RDEPEND-only deps we use a helper var like `COMMON_DEPEND` that both DEPEND and RDEPEND can inherit from.

Reply:
Ok! :) Will change this. However I suppose you had a typo in: `/usr/$(get_libdir)/news/$(get_libdir)` and you mean: `/usr/$(get_libdir)/news/bin/rnews` ?

Reply:
yes that must have been a copy-paste error, I just looked at `inn` ebuild without having it installed